src/app/admin/AdminPage.tsx

```diff
@@ -44,7 +66,7 @@ const AdminPage: React.FC<Props> = (props) => {
                     <TableSurface key={index}>
                         <Accordion elevation={0}>
                             <AccordionSummary expandIcon={<ExpandMore />}>
-                                <PanelIcon size="2x" icon={panelIcon} />
+                                <StyledIcon size="2x" icon={panelIcon} />
```

Comment:
why was this renamed from PanelIcon to StyledIcon? I think PanelIcon makes more sense - feel free to just change the CSS if you prefer 0.9rem of padding!

Reply:
Mmm okie!